### ISSUE TITLE
Fix error with __repl__ calculation for _ with getattr 

### ISSUE THREAD

**Issue by kachayev** [kachayev/fn.py#22]
``` python
>>> from fn import _ as XXX
>>> from fn.op import flip
>>> print flip(XXX + XXX)
(x1, x2, x3, x4, x5, x6) => getattr((x1 + x2), x3x4flipbackx5x6)
```

**Comment by KangOl** [comment]
I don't have the same output.

```
>>> from fn.op import flip
>>> from fn import _ as X
>>> flip(X+X)
<function _flipper at 0x10f789758>
>>> str(flip(X+X))
'<function _flipper at 0x10f78d488>'
```

**Comment by kachayev** [comment]
Oh, this was another issue, that I have already fixed today (#23).
Another example:

``` python
>>> from fn import _ as X
>>> X + X
(_ + _)
>>> (X + X).now
getattr((_ + _), now)
>>> str(_)
'(x1, x2) => getattr((x1 + x2), now)'
>>> (X + X).__and_now__
getattr((_ + _), __and_now__)
>>> str(_)
'(x1, x2, x3, x4, x5, x6, x7) => getattr((x1 + x2), x3x4andx5nowx6x7)'
```

**Comment by KangOl** [comment]
Ok, I see what the error is. I have a patch for it.
Pull request soon (but will depend on another one)
